Scripts/Runtime/Entities/Tasks/Attributes/ResolveChannelAttribute.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+
+namespace Anvil.Unity.DOTS.Entities
+{
+    [AttributeUsage(AttributeTargets.Field, AllowMultiple = true)]
```

Comment:
`AttributeTargets.Property` is ok too?

Reply:
As of today, no. I've added a TODO to expand support for it. 

Reflection is just easier to do on fields so I did that first.